bin/node-template/pallets/template/src/mock.rs

```diff
@@ -40,7 +40,7 @@ impl system::Trait for Test {
 	type Version = ();
 	type ModuleToIndex = ();
 	type AccountData = ();
-	type OnNewAccount = ();
+	type MigrateAccount = (); type OnNewAccount = ();
```

Comment:
This pattern repeats over multiple files. Looks like some broken s&r

Reply:
yeah, my editor doesn't properly support multiline s&r. but since all of this code will be removed 24 hours from now (and i know it's a trivial operation for my editor as long as there's no multiline), i don't really see the point in trying to fix it.

Reply:
k